delayed_job_active_record.gemspec

```diff
@@ -1,5 +1,5 @@
 Gem::Specification.new do |spec|
-  spec.add_dependency "activerecord", [">= 3.0", "< 5.3"]
+  spec.add_dependency "activerecord", [">= 3.0", "<= 6"]
```

Comment:
We should just remove this upper limit as there aren't any known issues with 6.x that we want to protect against (and right now, the `=` bit is not allowing testing alphas).